Stellar-transaction.x

```diff
@@ -1876,7 +1876,8 @@ enum InvokeHostFunctionResultCode
     INVOKE_HOST_FUNCTION_TRAPPED = -2,
     INVOKE_HOST_FUNCTION_RESOURCE_LIMIT_EXCEEDED = -3,
     INVOKE_HOST_FUNCTION_ENTRY_ARCHIVED = -4,
-    INVOKE_HOST_FUNCTION_INSUFFICIENT_REFUNDABLE_FEE = -5
+    INVOKE_HOST_FUNCTION_INSUFFICIENT_REFUNDABLE_FEE = -5,
+    INVOKE_HOST_FUNCTION_INVALID_PROOF = -6
```

Comment:
nit: `INVALID_RESTORATION_PROOF` or something along these lines? Just 'proof' is ambiguous.